app/controllers/users_controller.rb

```diff
@@ -1,4 +1,24 @@
 class UsersController < ApplicationController
   def new
+  	@user = User.new
   end
+
+  def show
+    @user = User.find params[:id]
```

Comment:
1. What will happen if it don't find the user with params[:id] 
2. Can you study about find_by and find.

Reply:
If it don't find the user with params[:id] , It will raises an Exception : ActiveRecord::RecordNotFound and server is crashed 
find_by will return nil in this case.

Reply:
Is it good if the application throws exception here?
1. If you use find, you have to rescue the exception
2. You can use find_by to prevent the application from throwing exception and then implement your own logic about what the application should do if the user is nil?

Reply:
I learned that I should use find_by to implement logic in show.html like If @user is nil, it will show error message like  can't find user by this id.